game_manager.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'game'
+require_relative 'author'
+require 'json'
+require_relative './data_storage_handler'
+
+class GameManager
+  attr_accessor :games
+
+  def initialize
+    @games = []
+  end
+
+  def add_game(author_manager)
+    puts 'Multiplayer? (y/n)'
+    multi_player = gets.chomp
+    puts 'Last played at (yyyy-mm-dd)'
+    last_played_at = gets.chomp
+    puts 'Publish date (yyyy-mm-dd)'
+    publish_date = gets.chomp
```

Comment:
[Optional]
It'd be great if you can add a Check if the user inputs correct Date Format So the program doesn't crash.

Reply:
I don't know how

Reply:
No Problem, it's optional